src/routes/(infoPages)/about/events/event/[dateAndSlug]/+page.server.ts

```diff
@@ -0,0 +1,98 @@
+import { error } from "@sveltejs/kit";
+import gql from "graphql-tag";
+import { print as printQuery } from "graphql";
+import { CONTENTFUL_DELIVERY_API_TOKEN, CONTENTFUL_SPACE_ID } from "$env/static/private";
+import { day } from "$lib/constants/date";
+import getContentfulClient from "$lib/services/contentful";
+import type { PageServerLoad } from "./$types";
+import type { EventQuery } from "./$queries.generated";
+import { loadBaseBreadcrumbs } from "../../shared.server";
+
+const query = gql`
+  query Event($dateStart: DateTime!, $dateEnd: DateTime!, $slug: String!) {
+    eventEntryCollection(
+      limit: 1
+      where: { eventDateAndTime_gte: $dateStart, eventDateAndTime_lt: $dateEnd, slug: $slug }
+    ) {
+      items {
+        sys {
+          id
+        }
+        slug
+        shortTitle
+        internalName
+        eventDescription
+        eventDateAndTime
+        eventDocumentsCollection {
+          items {
+            __typename
+            ... on DocumentWrapper {
+              sys {
+                id
+              }
+              docWrapperName
+              documentDescription
+              wrappedDocumentName {
+                url
+              }
+            }
+            ... on Contact {
+              sys {
+                id
+              }
+              entityName
+              phone
+              phoneExt
+              email
+              location {
+                name
+                streetAddress1
+                streetAddress2
+                city
+                state
+                zip
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`;
+
+export const load = (async ({ params: { dateAndSlug }, parent }) => {
+  // dateAndSlug should be constructed like 2023-08-10-some-slug
+  const [_, dateString, slug] = dateAndSlug.match(/(\d{4}-\d{2}-\d{2})-([a-z1-9-]+)/) ?? [];
+  if (!dateString || !slug) throw error(404);
+  const date = new Date(dateString);
+  const dateStart = date.toISOString();
+  const dateEnd = new Date(date.getTime() + 1 * day).toISOString();
```

Comment:
Not important for right now, and I haven't looked into this much myself... I know we're defaulting to central time (`UTC-05:00`) in Contentful, but is there anything we need to do in the rare case that a different timezone is used? Do we know if daylight savings time will be accounted for?